app/controllers/students_controller.rb

```diff
@@ -0,0 +1,80 @@
+class StudentsController < ApplicationController
+  
+  def index
+    @students = Student.all
+    @s_classes =StudentClass.all
+  end
+
+  def show
+    # binding.pry
+    @student = Student.find_by_id(params[:id])
+  end
+
+  def new
+    # binding.pry
+    @student = Student.new({:name => 'Enter your name', :father_name => 'Enter your Father Name ', age:"18"})
```

Comment:
@student = Student.new 

why you are passing these values of ENTER your Name

Reply:
done